apps/nextjs-example/src/components/WalletSelector.tsx

```diff
@@ -86,33 +99,55 @@ interface ConnectWalletDialogProps {
 }
 
 function ConnectWalletDialog({ close }: ConnectWalletDialogProps) {
-  const { wallets } = useWallet();
-  const { defaultWallets, moreWallets } = partitionWallets(wallets ?? []);
+  const { wallets = [] } = useWallet();
+  const { aptosConnectWallets, otherWallets } = getAptosConnectWallets(wallets);
+  const { defaultWallets, moreWallets } = partitionWallets(otherWallets);
```

Comment:
I find the naming a bit confusing `aptosConnectWallets`, `otherWallets`, `defaultWallets`, `moreWallets` - can we at least leave comments explaining what each means?